ch.tlaun.TL.appdata.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id type="desktop">ch.tlaun.TL.desktop</id>
```

Comment:
```suggestion
    <id type="desktop">ch.tlaun.TL</id>
```

the app-id shall not end with .deskop.

Reply:
Actually it should, otherwise flatpak-builder will fail with error.

Reply:
I doubt it fail because of that.

I must insist.

```suggestion
    <id>ch.tlaun.TL</id>
```

Reply:
Okay, I wrongly checked `<launchable>` tag, sorry.
Fixed!